nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/xml/inference/XmlRecordSource.java

```diff
@@ -132,7 +134,7 @@ private XmlNode readNext(final StartElement startElement) throws XMLStreamExcept
         } else {
             final String textContent = content.toString().trim();
             if (!textContent.equals("")) {
-                childNodes.put("value", new XmlTextNode("value", textContent));
+                childNodes.put(contentFieldName, new XmlTextNode(contentFieldName, textContent));
```

Comment:
If "content field name" property is not set, the value is still lot. I would suggest to extend the description of the propery with information about this.

Reply:
That is right, thank you. I extended the property description and added detailed examples to the "Additional Details..." part of the documentation.